help/en/html/doc/Technical/Help.shtml

```diff
@@ -28,14 +19,34 @@
     <!--#include virtual="Sidebar.shtml" -->
     <div id="mainContent">
 
-      <h1>JMRI Code: Use of JavaHelp</h1>
-      
-      This page talks about technical
-      aspects of how JMRI provides help information using <a href=
-      "https://javahelp.java.net">JavaHelp</a> and <a href=
-      "http://jhelpdev.sourceforge.net/">JHelpDev</a>.      
-      
-      <h2>Organization of the help files</h2>The help files are
+<!-- = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = --> 
+     <a name="helpTop" id="helpTop"></a>     
+     <h1>JMRI Code: Creating Help Pages and Using JavaHelp</h1>
+
+    <div class="para">      
+     <p>This page talks about technical aspects of how JMRI provides help information 
+     using <a href="https://javahelp.java.net">JavaHelp</a> and <a href=
+      "http://jhelpdev.sourceforge.net/">JHelpDev</a> as well as some suggestions on 
```

Comment:
Project link has changed, please update to https://sourceforge.net/projects/jhelpdev/

Reply:
Change made.  Thanks.